grails-app/conf/Config.groovy

```diff
@@ -221,7 +221,8 @@ portal {
     siteHeader = "Open Access to Ocean Data"
     motdUrl = "https://static.emii.org.au/motd"
 
-    logo = "images/IMOS_logo.png"
+    logo = "images/AODN/AODN_logo.png"
+    secondaryLogo ="https://static.emii.org.au/images/logo/IMOS_logo-stacked3.png"
```

Comment:
This AODN/IMOS stuff is for development and wont be included in a production war. I think its useful to list here all the config items that are used by the portal. Its like a template for external sysadmins to modify and inject and of course for eMII developers its essential.